tests/integ/data/schema-tag-property-createonly.json

```diff
@@ -0,0 +1,16 @@
+{
+    "properties": {
+        "Tags": {},
+        "Arn": {}
+    },
+    "createOnlyProperties": [
+        "/properties/Tags"
+    ],
+    "tagging": {
+        "taggable": true,
+        "tagOnCreate": true,
```

Comment:
can you please add another case when Tags are not createOnly and tagUpdatable is set to true, then Tag018 should not fail right?

Reply:
Added additional test case